modules/messages/client/components/QuickReply.js

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export default function QuickReply({ onSend, onFocus }) {
+  const { t } = useTranslation('messages');
+
+  // It's not quite clear to me how to handle the i18n of
+  // these messages. They are a bit of a hack to start with.
+  // They rely on the "data-hosting" attribute being set on the
+  // p element, but it still has message content that will not
+  // be translated later, it's stuck at it's original content.
+  // I chose to write the message content in the language the
+  // user has set.
+  // Probably better would be not to include any message content,
```

Comment:
I'd be okay removing this feature for now too; we could analyze data a bit.

Initial motivation was to improve reply rates by making it easy to quickly reply with a click of a button (including from emails).

Reply:
I think it's OK to keep it in place, it's not _more_ hacky now than it was, and could be built on...  i.e. implementing "special messages" of the same type, and can still rewrite the old ones... but for another PR.

Reply:
Sounds good! 👍